client/public/index.html

```diff
@@ -27,6 +27,8 @@
     <link rel="icon" sizes="192x192" href="%PUBLIC_URL%/assets/images/misc/favicon.png" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/assets/images/misc/home-icon.png" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/assets/images/misc/home-icon.png" />
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.4/dist/leaflet.css" integrity="sha512-puBpdR0798OZvTTbP4A8Ix/l+A4dHDD0DGqYW6RQ+9jxkRFclaxxQb/SJAWZfWAkuyeQUytO7+7N4QKrDh+drA=="
+          crossorigin="" />
```

Comment:
I would prefer if the CSS wasn't imported here.
I noticed there is a lot of weirdness with importing the leaflet styling directly in the component.
A solution that works: https://github.com/PaulLeCam/react-leaflet/issues/255#issuecomment-388492108

Reply:
Would look into it.